modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/import.jsp

```diff
@@ -34,9 +34,11 @@ renderResponse.setTitle(LanguageUtil.get(resourceBundle, "import"));
 		<aui:input name="mvcPath" type="hidden" value="/admin/import.jsp" />
 		<aui:input name="parentKBFolderId" type="hidden" value="<%= String.valueOf(parentKBFolderId) %>" />
 
-		<liferay-ui:error exception="<%= KBArticleImportException.MustHaveACategory.class %>">
-			<%= LanguageUtil.format(request, "an-unexpected-error-occurred-while-importing-articles-x", LanguageUtil.get(request, "there-are-one-or-more-mandatory-vocabularies-assigned-to-the-knowledge-base-article")) %>
-		</liferay-ui:error>
+		<div class="mt-4">
+			<liferay-ui:error exception="<%= KBArticleImportException.MustHaveACategory.class %>">
+				<%= LanguageUtil.format(request, "an-unexpected-error-occurred-while-importing-articles-x", LanguageUtil.get(request, "there-are-one-or-more-mandatory-vocabularies-assigned-to-the-knowledge-base-article")) %>
+			</liferay-ui:error>
+		</div>
```

Comment:
Just a question, not a change request: why are we styling this error message in a different way? Shouldn't all error message look the same?

Reply:
In this case the error message it's the same, the only thing that changes is it's possition on the page